<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup_view</class>
 <widget class="QDialog" name="setup_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>952</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <widget class="QLabel" name="operation_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Operations</string>
   </property>
  </widget>
  <widget class="QLabel" name="data_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="cluster_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cluster</string>
   </property>
  </widget>
  <widget class="QLabel" name="prefactor_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Prefactor path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hdf5root_input">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>580</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="querying_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
  </widget>
  <widget class="QLabel" name="retrive_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Retrieve</string>
   </property>
  </widget>
  <widget class="QLabel" name="stage_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stage</string>
   </property>
  </widget>
  <widget class="QLabel" name="process_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="QLabel" name="which_obj_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Which obj</string>
   </property>
  </widget>
  <widget class="QComboBox" name="querying_comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>102</width>
     <height>36</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>True</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>True</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>False</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="retrieve_combobox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <item>
    <property name="text">
     <string>False</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>True</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="which_combobox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>calibrators</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>target</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>all</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="process_combobox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>False</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>True</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="stage_combobox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>False</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>True</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="WorkingPath_input">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="calibratorSASids_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="calibratorSASids_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrator SAS id</string>
   </property>
  </widget>
  <widget class="QLabel" name="product_type_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="PROJECTid_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Project id</string>
   </property>
  </widget>
  <widget class="QLabel" name="Target_name_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetSASids_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Target SAS id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Target_name_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="targetSASids_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="PROJECTid_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="product_type_combobox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>141</width>
     <height>36</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>observation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pipeline</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="max_per_node_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max per node</string>
   </property>
  </widget>
  <widget class="QLabel" name="method_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Method</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_per_node_input">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="method_input">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="wsclean_executable_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>/opt/cep/wsclean/wsclean-2.8/bin/wsclean</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="losotoPath_input">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>780</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>/data/scratch/iacobelli/losoto_Nov21_latest_commit_a7790a6/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pythonpath_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>790</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="task_file_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>890</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>%(lofarroot)s/share/pipeline/tasks.cfg</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lofarroot_input">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>/opt/cep/lofim/daily/Tue/lofar_build/install/gnucxx11_opt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PrefactorPath_input">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="casaroot_input">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>/opt/cep/casacore/casacore_current</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="aoflagger_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>/opt/cep/aoflagger/aoflagger-2.10.0/build/bin/aoflagger</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wcsroot_input">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>680</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pyraproot_input">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>431</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>/opt/cep/casacore/python-casacore_current/lib64/python2.7/site-packages</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Paths</string>
   </property>
  </widget>
  <widget class="QLabel" name="task_file_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>850</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>task_file</string>
   </property>
  </widget>
  <widget class="QLabel" name="losotoPath_label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>740</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>losotoPath</string>
   </property>
  </widget>
  <widget class="QLabel" name="wcsroot_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>640</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>wcsroot</string>
   </property>
  </widget>
  <widget class="QLabel" name="hdf5root_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>540</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>hdf5root</string>
   </property>
  </widget>
  <widget class="QLabel" name="pyraproot_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pyraproot</string>
   </property>
  </widget>
  <widget class="QLabel" name="casaroot_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>casaroot</string>
   </property>
  </widget>
  <widget class="QLabel" name="lofaroot_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>lofarroot</string>
   </property>
  </widget>
  <widget class="QLabel" name="working_path_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Working path</string>
   </property>
  </widget>
  <widget class="QLabel" name="pythonpath_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>750</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pythonpath</string>
   </property>
  </widget>
  <widget class="QLabel" name="wsclean_executable_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>wsclean executable</string>
   </property>
  </widget>
  <widget class="QLabel" name="aoflagger_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>aoflagger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>890</y>
     <width>221</width>
     <height>36</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>226</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>198</red>
         <green>255</green>
         <blue>143</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>168</red>
         <green>240</green>
         <blue>97</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>113</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>92</red>
         <green>151</green>
         <blue>34</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>226</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>196</red>
         <green>240</green>
         <blue>153</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>226</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>198</red>
         <green>255</green>
         <blue>143</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>168</red>
         <green>240</green>
         <blue>97</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>113</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>92</red>
         <green>151</green>
         <blue>34</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>226</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>196</red>
         <green>240</green>
         <blue>153</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>113</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>226</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>198</red>
         <green>255</green>
         <blue>143</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>168</red>
         <green>240</green>
         <blue>97</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>113</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>92</red>
         <green>151</green>
         <blue>34</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>113</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>113</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>226</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>226</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>138</red>
         <green>226</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Save LAnDmARk configuration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
